aQute.libg/test/aQute/libg/uri/URIUtilsTest.java

```diff
@@ -1,56 +1,125 @@
 package aQute.libg.uri;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.condition.OS.WINDOWS;
+
 import java.net.URI;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
 
-import aQute.lib.io.IO;
-import junit.framework.TestCase;
+import org.junit.jupiter.api.Test;
```

Comment:
I guess it would be nice if this PR did not depend on the Junit 5 PR. Then it is held out waiting for that to be integrated.

Reply:
I anticipated that the JUnit 5 PR would get merged before this one (as indeed it has).